sdk/tests/deqp/functional/gles3/es3fNegativeBufferApiTests.js

```diff
@@ -564,22 +564,23 @@ goog.scope(function() {
                 gl.checkFramebufferStatus(gl.FRAMEBUFFER);
                 this.expectError(gl.NO_ERROR);
 
-                bufferedLogToConsole('gl.INVALID_ENUM is generated if one of the values in bufs is not an accepted value.');
-                gl.drawBuffers(values.splice(2, 2));
-                this.expectError(gl.INVALID_ENUM);
+                bufferedLogToConsole('gl.INVALID_OPERATION is generated if the GL is bound to framebuffer object and one of the values in bufs is not an accepted value.');
+                gl.drawBuffers(values.slice(0, 2));
+                this.expectError(gl.INVALID_OPERATION);
 
-                bufferedLogToConsole('gl.INVALID_OPERATION is generated if the GL is bound to the default framebuffer and n is not 1.');
-                gl.drawBuffers(values.splice(0, 2));
+                bufferedLogToConsole('gl.INVALID_OPERATION is generated if the GL is bound to the default framebuffer and the number of queried buffers is not 1.');
```

Comment:
Ken, the drawBuffers() API in WebGL has no argument 'n', this is a little different from the native one. So I revised the log.